ERCS/erc-7656.md

```diff
@@ -131,9 +149,28 @@ interface IERC6551Account {
 
 or both. This flexibility makes [ERC-6551](./eip-6551.md) accounts compatible with this proposal out-of-the-box.
 
-_Notice that most likely the function state will return a bytes32 value in the final version of [ERC-6551](./eip-6551.md). As soon as that changes, we will update this proposal accordingly._
+_Notice that most likely the function `state` in `IERC6551Account` will return a `bytes32` value in the final version of [ERC-6551](./eip-6551.md). As soon as that changes, we will update this proposal accordingly._
```

Comment:
Write proposals as if they were already in Final. This line wont make sense in the future.

Reply:
Ok. I will remove the comment and when they will update ERC6551 I will fix it.

Reply:
I actually removed the details of the interface.